src_teal/texrunner/checkdriver.tl

```diff
@@ -1,22 +1,8 @@
---[[
-  Copyright 2020 ARATA Mizuki
-  Copyright 2024 Lukas Heindl
+-- SPDX-FileCopyrightText: 2020 ARATA Mizuki
+-- SPDX-FileCopyrightText: 2024 - 2025 Lukas Heindl
+--
+-- SPDX-License-Identifier: GPL-3.0-or-later
 
-  This file is part of CluttealTeX.
-
-  CluttealTeX is free software: you can redistribute it and/or modify
-  it under the terms of the GNU General Public License as published by
-  the Free Software Foundation, either version 3 of the License, or
-  (at your option) any later version.
-
-  CluttealTeX is distributed in the hope that it will be useful,
-  but WITHOUT ANY WARRANTY; without even the implied warranty of
-  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-  GNU General Public License for more details.
-
-  You should have received a copy of the GNU General Public License
-  along with CluttealTeX.  If not, see <http://www.gnu.org/licenses/>.
-]]
```

Comment:
@minoki can you please check if this works for you as well? If not, I'll keep your license block and just put the new `SPDX` comments on top (thought having this in a uniform way would be nice).

Not sure how much in detail you look at the diff. In case you do, don't wonder about references to you in files you don't know, I moved stuff around a bit and since you're still the author of that functions (I just copied at that point), I wanted to keep the attribution.